### DISCUSSION TITLE
Xdrip watch face on the smartwatch

### DISCUSSION THREAD

**Discussion by stefanotoppounipd** [NightscoutFoundation/xDrip#3497] [General]
Hi all,
Usually, to install the watch face on my smartwatch (TicWatch Pro 3 GPS, now updated to Wear OS 3.5), I extract the APK (android_wear_micro_apk.apk) from the APK of the latest xDrip release. This APK is usually in the res/raw directory. In the current latest release [Nightly build 19th Apr 2024], I cannot find this Wear OS APK. As a workaround, I have reinstalled the new latest xDrip release and the old android_wear_micro_apk.apk that I extracted from the Nightly build 16th Nov 2022.
Is it possible to have this watch face back in the upcoming releases of xDrip? It is a very useful watch face for those like me who wear a smartwatch to monitor blood sugar with Libre 2.
Thanks!

**Comment by PhilTU** [comment]
I was digging around in this issue and the apk is now in res with a different name sq.apk see https://github.com/NightscoutFoundation/xDrip/discussions/3480 

What I don't know yet is whether you can install it from there or if Wear Installer 2 will pull it from there.

**Reply by stefanotoppounipd** [reply]
Hi,
Thank you very much! I wasn't aware of these changes. Indeed, sq.apk is rather "cryptic." I wonder why this part is so poorly documented now. I've also read your thread. Currently, I force the installation with ADB rather than using Wear Installer 2, which I used in the past.
All the best

**Comment by stefanotoppounipd** [comment]
By the way, instead of using WinZip, 7-zip, etc., to uncompress .apk files, I've tried apktool and it seems to solve the issue. The latest release of Xdrip (after being uncompressed by apktool) shows the expected directory tree and in ./res/raw there is the android_wear_micro_apk.apk!

**Comment by PhilTU** [comment]
thanks for the tip about using apktool
